src/markdown-pages/add-ons/rook.md

```diff
@@ -78,3 +78,7 @@ spec:
 It is not possible to upgrade multiple minor versions of the Rook add-on at once.
 Individual upgrades from one version to the next are required for upgrading multiple minor versions.
 For example, to upgrade from `1.4.3` to `1.6.11`, you must first install `1.5.10`, `1.5.11` or `1.5.12`.
+
+For online installations, you can upgrade from Rook `1.0.x` to `1.4.9` with `curl https://k8s.kurl.sh/latest/tasks.sh | sudo bash -s rook_10_to_14`.
+This will migrate data off of any hostpath-based OSDs in favor of block device-based OSDs and upgrade through Rook `1.1.9`, `1.2.7` and `1.3.11` before installing `1.4.9`.
```

Comment:
is it possible to reword this a bit to make it clear that this is the only supported method for upgrades from rook 1.0.4?

Reply:
Sure